volatility3/cli/__init__.py

```diff
@@ -500,6 +500,16 @@ def run(self):
                 renderer.filter = text_filter.CLIFilter(grid, args.filters)
                 renderer.column_hide_list = args.hide_columns
                 renderer.render(grid)
+        except exceptions.UnsatisfiedException as excp:
+            output = sys.stderr
```

Comment:
Why not just put a new entry in the `process_exceptions` handler?  We really don't need to be doing the full requirement/unsatisfied stuff, this is only when there's a mismatch between the framework old method and the framework new method...